pyvista/plotting/actor_properties.py

```diff
@@ -0,0 +1,149 @@
+"""Module containing pyvista implementation of vtkProperty."""
+from enum import Enum
+
+import pyvista as pv
+
+
+class ShaderModel(Enum):
+    """Types of shader methods available."""
+
+    FLAT = 0
+    GOURAUD = 1
+    PHONG = 2
+    PBR = 3
+
+
+class ReprModel(Enum):
+    """Types of representations the models can have."""
+
+    POINTS = 0
+    WIREFRAME = 1
+    SURFACE = 2
+
+
+class ActorProperties:
```

Comment:
We might want to leverage the default colors for the active theme here. I expect the text color to match that of the current theme by default, but it doesnt

Reply:
I used the active theme color in `AxesActor` class, since each axis should have a different color in the theme. 
https://github.com/pyvista/pyvista/blob/225ac8a866dcf52a9a04d6b9cd061e247bb192ba/pyvista/plotting/axes_actor.py#L69-L89